gammapy/modeling/models/tests/test_temporal.py

```diff
@@ -187,6 +194,14 @@ def test_gaussian_temporal_model_integral():
     assert_allclose(np.sum(val), 0.682679, rtol=1e-5)
 
 
+def test_gaussian_temporal_model_sample_time():
+    t_ref = Time(55555, format="mjd")
+    temporal_model = GaussianTemporalModel(t_ref=t_ref.mjd * u.d, sigma="2.0 day")
+    times = temporal_model.sample_time(2)
+
+    assert_allclose(times.value, [55555.00000291, 55554.99999694], rtol=1e-8)
```

Comment:
the difference in the test suggest that the sigma is not 2 days. Maybe an issue with the unit propagation in `sample_time`?